automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl

```diff
@@ -393,3 +393,234 @@ groups:
       summary: "Missing blocks observed on {{ $labels.testnet }}"
```

Comment:
I think you need to remove `berkeley` from `$labels.testnet`?

Reply:
I didn't have every alerts in that group, for example I didn't put NodesOutOfSync in this group and I didn't put MissingBlocksObserved or any watchdog related alerts. I think we still need those alerts for berkeley.

Reply:
But then what effect would the ones with sync status have on our alerting? The duplicated alerts without the sync status will keep firing for berkeley

Reply:
I'd suggest duplicating them for now for simplicity and when these are sufficiently tested and mainnet has syncStatus label they'll be deduplicated.

Reply:
I didn't find where $labels.testnet is defined, it's not one of the local variable and not one of the input. I modify the `rule_filter` to filter out berkeley instead.